### PR TITLE
BC-329 - Feature flag: button to nuxt school admin

### PR DESCRIPTION
**Pull request by JohnKoumarelas**
# Description
Adds feature flag to the new Nuxt School admin page.

## Links to Tickets or other pull requests
https://ticketsystem.hpi-schul-cloud.org/browse/BC-329

## Approval for review

- [x] QA: In addition to review, the code has been manually tested (if manual testing is possible)
- [x] All points were discussed with the ticket creator, support-team or product owner. The code upholds all quality guidelines from the PR-template.

> Notice: Please remove the WIP label if the PR is ready to review, otherwise nobody will review it.

### Link to Definition of Done
More and detailed information on the *definition of done* can be found [on Confluence](https://docs.schul-cloud.org/pages/viewpage.action?pageId=92831762)

### CODE REVIEW

**Comment by JohnKoumarelas** [comment]
We also need FEATURE_NUXT_SCHOOL_ADMIN_BETA in ConfigMap.

**Review by murat-merdoglu-hpi — DISMISSED** [review]

File: config/global.js [hunk 1]
```diff
@@ -82,6 +83,7 @@ const exp = {
 	SC_SUPERHERO_USER_PASSWORD,
 	SC_SUPERHERO_USER_NAME,
 	HELPAREA_URL,
+	FEATURE_NUXT_SCHOOL_ADMIN_BETA,
```
Comment (CeEv) [inline config/global.js:86]: default.schema.json are the better place but this is ok as temporary solution.

Comment (CeEv) [reply config/global.js:86]: Ah ok it is alread in default.schema.json. Than it should not nessasrary to add it to global too.

**Comment by CeEv** [comment]
Please add the changelog entry.

**Review by murat-merdoglu-hpi — APPROVED** [review]

**Pull request merged**